extensions/wikia/Wall/index/CommentsIndex.class.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+/**
+ * CommentsIndex Class
+ * @author Kyle Florence, Saipetch Kongkatong, Tomasz Odrobny
+ */
+
+class CommentsIndex {
```

Comment:
Can we remove `extensions/wikia/ArticleComments/classes/CommentsIndex.class.php` file? According to GitHub's diff it still exists.